core/collector.class.inc.php

```diff
@@ -14,6 +14,8 @@
 //   You should have received a copy of the GNU Affero General Public License
 //   along with this application. If not, see <http://www.gnu.org/licenses/>
 
+require_once(APPROOT.'core/callitopservice.class.inc.php');
+
```

Comment:
Is this the correct place to add this?

Reply:
why not? it seems simpler to me to include it only once here. instead of all the places where collector.php is required also (exec.php + all the tests).

Reply:
Yes, that is also done for `ioexception.class.inc.php` and similar.

Reply:
Still no response..

Reply:
IOException class should be found everywhere this kind of exception is thrown and catched. 

Regarding CallItopService this class is used  only within dedicated test and collector.php file. that is why I did not spread this require everywhere. it seems cleaner to me. 
collector.php is spread already. it does the job for callitopservice.class.inc.php as well. no?

Reply:
It's fine for me if others in the team agree with this added inconsistency.

Reply:
It is fine by me as well to leave this require here as it is the only place it is used. That's why I didn't comment before.